wiki/Interface/en.md

```diff
@@ -1,227 +1,297 @@
+7. The mods that you have activated (see #12), as well as the option to select them. The option marked "Free Mods" toggles whether or not players can select their own mods. If yes, they can pick any combination of mods *except for speed-altering mods like Double Time*. If no, the host decides what mods will be used. The host can pick speed-altering mods regardless of whether or not Free Mods is turned on.
+8. The team mode and win conditions. 
+9. The ready button.
+10. The chat box.
+11. The leave button. 
+12. Where your activated mods appear.
 
-Song Selection Screen
----------------------
+## Song Selection Screen
 
-<img src="soloplay1.jpeg" title="Song selection" alt="Song selection" width="300" />
+![Song Selection](/wiki/Interface/img/Soloplay1.jpeg)
 
-Note: You can identify the song selection for a specific mode by checking the transparent leitmotif.
+Note: You can identify the current mode selected by either looking at the icon in the bottom right, above Mode, or by looking at the transparent icon in the center of the screen.
 
-![](osu.gif "fig:osu.gif")"osu!" is [osu!](Standard "wikilink"), ![](taiko.gif "fig:taiko.gif")"Drum" is [Taiko](Taiko "wikilink"), ![](ctb.gif "fig:ctb.gif")"Apple" is [Catch the Beat](Catch_the_Beat "wikilink") and ![](mania.gif "fig:mania.gif")"Piano" is [osu!mania](Mania "wikilink")
+![][Osu!]"osu!" is [osu!](Standard "wikilink"), ![][Taiko]"Drum" is [Taiko](Taiko "wikilink"), ![][Ctb]"Apple" is [Catch the Beat](Catch_the_Beat "wikilink") and ![][Mania]"Piano" is [osu!mania](Mania "wikilink")
 
-Because this screen has too many elements to note by easily noticeable numbers, this section will focus on one part of the screen at a time, starting from the top (left to right) and continuing downwards.
+Because this screen has too many elements to note by easily noticeable numbers, this section will focus on one part of the screen at a time, starting from the top down and left to right.
 
 ### Beatmap Information
 
-<img src="soloplay2.jpeg" title="fig:soloplay2.jpeg" alt="soloplay2.jpeg" width="400" />
-This area displays **information on the beatmap difficulty currently selected.** By default, the beatmap whose song is heard in the osu! jukebox is selected when entering the selection screen. The BPM value listed is the deviation between highest and lowest BPM that occurs in the beatmap (timing section changes). "Objects" is the total number of [Hit Objects](Hit_Objects "wikilink") in the beatmap. A ![](heart.gif "fig:heart.gif")heart symbol at the upper left denotes a Ranked beatmap while a ![](fire.gif "fig:fire.gif")flame symbol denotes Approved beatmap. The Unicode title will be used if [allowed non-English metadata at Options under General section](Options#Language "wikilink") and the beatmap itself has Unicode title/artist.
+![soloplay2.png](/wiki/Interface/img/Soloplay2.jpeg)
+![soloplay2.5.png](/wiki/Interface/img/Soloplay2.5.jpeg)
```

Comment:
Sorry, Soloplay2.5 was for some reason the same file. It's been updated to a different beatmap that has varying BPMs so that the romanization and BPM examples are visible.